portafly/doc/ux_specs/index.md

```diff
@@ -20,6 +20,8 @@ Please refer to the single page files together with mockups linked in each page
 * _coming soon_
 
 ##### Product
+* [Create product](./product/create_product.md)
```

Comment:
`create_product.md` file is missing

Reply:
This file is created in https://github.com/3scale/porta/pull/2030, better remove this line from this PR